src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -98,7 +98,9 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
 /**
  * Read attribute in a designated attribute path
```

Comment:
```suggestion
 * Read attribute in a designated attribute path.  If there is no value available
 * for the attribute, whether because the device does not implement it or
 * because the subscription priming read has not yet gotten to this attribute,
 * nil will be returned.
```